Update java.md (#8990)
DataDog/documentation: content/en/tracing/setup_overview/setup/java.md

   ```

     For more information, see the [Oracle documentation][4].
     

> - The `dd-java-agent` should never be added to your classpath and can cause unexpected behavoir. 

Reviewer: ```suggestion
- Never add `dd-java-agent` to your classpath. It can cause unexpected behavior. 
```
